### DISCUSSION TITLE
Welcome to react-native-audio-recorder-player Discussions!

### DISCUSSION THREAD

**Discussion by hyochan** [dooboolab/react-native-audio-recorder-player#281] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by StoychoMihaylov** [comment]
Hi, do you plan to add functionality for sound waveform ? I think if this could be added the player would be great!

**Comment by hyochan** [comment]
<img width="1428" height="933" alt="Screenshot 2025-07-20 at 6 41 44 PM" src="https://github.com/user-attachments/assets/2c88f580-4e2b-43f3-a177-bf19d2d40fd5" />
